recipes-kernel/linux/linux-mainline/arm/defconfig

```diff
@@ -956,7 +1040,6 @@ CONFIG_OF_KOBJ=y
 CONFIG_OF_ADDRESS=y
 CONFIG_OF_IRQ=y
 CONFIG_OF_NET=y
-CONFIG_OF_MDIO=y
```

Comment:
Think this is necessary for the Network PHY ..
Does the ethernet port work with this kernel config ?

Reply:
Yes, it works fine, as well  with the RJ45 connector as with USB ethernet adapter as with wifi adapter...

Reply:
ok